particle/particle/enums.py

```diff
@@ -43,12 +52,19 @@ class Status(IntEnum):
 
 
 # Mappings that allow the above classes to be produced from text mappings
-Par_mapping = {'+': Par.p, '0': Par.o, '+2/3': Par.u,
-               '++': Par.pp, '-': Par.m, '-1/3': Par.u, '?': Par.u, '': Par.o}
+Par_mapping = {'+': Par.p, '0': Par.o, '-': Par.u, '?': Par.u, '': Par.o}
+Charge_mapping = {'+': Charge.p, '0': Charge.o, '+2/3': Charge.p23,
```

Comment:
Why did you not add entries for -2/3 and +1/3? Unless I missed it.

Reply:
They are not used, but I added them anyway now just in case someone uses these for something else.

Reply:
Yep, better avoid confusion with not having them ;-). Thanks.